frontend/src/components/Landing/signin-button.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import Button from 'react-bootstrap/Button';
+
+import { SIGN_IN } from '../../constants/routes';
+
+/**
+ * SignInButton component used for the Landing page to redirect to the SignIn
+ * page.
+ */
+const SignInButton = () => {
+  const history = useHistory();
+
+  function goToSignIn() {
+    history.push(SIGN_IN);
+  }
+
+  return (
+    <Button type='button' onClick={goToSignIn} variant='primary' size='lg'>
+      Sign In
+    </Button>
+  );
+}
+
+export default SignInButton;
```

Comment:
One part is that SignInButton uses the `useHistory` function, which requires that it be used in  function components. If I didn't put it as a separate component, it would require me to make the entire Landing component a functional component.

Also, even though I could have declared the SignInButton component in the index.js file as well, I like having every file contain at most 1 component. I think it's generally considered cleaner this way?